lib/iris/tests/results/usecases/pp_to_cf_conversion/from_netcdf/008000000000.44.101.000128.1890.09.01.00.00.b_0.cml

```diff
@@ -18,10 +18,10 @@
         </dimCoord>
       </coord>
       <coord datadims="[1]">
-        <dimCoord id="6bc2e82a" points="[681540.0, 683340.0, 685140.0, 686940.0,
+        <dimCoord id="2789ad5b" points="[681540.0, 683340.0, 685140.0, 686940.0,
```

Comment:
Given the name of the netcdf file and 

```
>>> iris.unit.Unit('hours since 1970-01-01 00:00:00', calendar='360_day').num2date(681540.0)
2048-11-18 12:00:00
>>> iris.unit.Unit('days since 0000-01-01 00:00:00', calendar='360_day').num2date(681540.0)
1893-03-01 00:00:00
```

I'd say that the ``days since epoch`` was the correct unit here. That is not to say that this file is correct, just to say that there is some existing behaviour which is correct for some files, which your change breaks.

Does the spec give any guidance on the unit?